app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb

```diff
@@ -0,0 +1,99 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow < ManageIQ::Providers::AnsiblePlaybookWorkflow
+  def load_transitions
+    super.merge(:post_execute => {'running' => 'post_execute_poll'}, :post_execute_poll => {'post_execute_poll' => 'post_execute_poll'})
```

Comment:
Maybe it is just because I'm used to seeing these on multiple lines, but I find this harder to read

Can you split up like:
```ruby
super.merge(
  :post_execute      => {...},
  :post_execute_poll => {...}
)
```

Reply:
done